### PR TITLE
[release/v2.21] Updating changelog after latest fix

### PR DESCRIPTION
**Pull request by pkprzekwas**
**What this PR does / why we need it**:
Updating changelog after latest fix.

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind chore

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,8 +27,14 @@
 
 ### Updates
 
-- Update machine-controller to v1.54.5. This fixes an issue with Flatcar nodes not joining the cluster for clusters that don't use OSM ([#11897](https://github.com/kubermatic/kubermatic/pull/11897))
+- Update machine-controller to v1.54.5 ([#11897](https://github.com/kubermatic/kubermatic/pull/11897))
+    - Fix an issue with Flatcar nodes not joining the cluster for clusters that don't use OSM.
+- Update Operating System Manager to v1.2.1. ([#12049](https://github.com/kubermatic/kubermatic/pull/12049))
```
Comment (hdurand0710) [inline CHANGELOG.md]: For 2.21, it's OSM 1.1.3 (PR 12050)

**Review by hdurand0710 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pkprzekwas) [reply CHANGELOG.md]: Oh, thanks for being watchful :D

**Comment by pkprzekwas** [comment]
@hdurand0710 can you un-hold?

**Pull request merged**